frontend/Movie_Pigeon/app/src/main/java/org/example/team_pigeon/movie_pigeon/MoviePageFragment.java

```diff
@@ -230,7 +227,7 @@ private void setTextOtherwiseSetGone(String item, TableRow row, TextView text) {
     }
 
     private void setRatingsOtherwiseSetGone(String rating, TextView score, ImageView icon) {
-        if (rating != null) {
+        if (rating != null && !rating.equals("0")) {
```

Comment:
It seems that a rating of 0 is possible. This can be achieved by rate something non-zero first and then change it to zero, no?